src/bootstrap-table.js

```diff
@@ -156,6 +158,10 @@ class BootstrapTable {
         }, $(el).data()))
       })
       columns.push(column)
+
+      if ($(el).attr('class')) {
+        this._headerTrClasses.push($(el).attr('class'))
```

Comment:
Could we save the $(el) object in a const variable? This is in order to avoid multiple creation of jquery objects.